jobs/build/operator-sdk_sync/Jenkinsfile

```diff
@@ -15,7 +15,7 @@ node {
                 commonlib.ocpVersionParam('BUILD_VERSION'),
                 string(
                     name: 'ASSEMBLY',
-                    description: 'Under which ASSEMBLY directory to place the binaries.',
+                    description: 'Under which ASSEMBLY directory to place the binaries. If not provide will use latest release in candidate channel.',
```

Comment:
"If empty, then the latest release in candidate channel would be used as the assembly."